pkg/cmd/cli/cmd/rsync/copymulti.go

```diff
@@ -26,13 +26,14 @@ func (ss copyStrategies) Copy(source, destination *pathSpec, out, errOut io.Writ
 		err = s.Copy(source, destination, out, errBuf)
 		if _, isSetupError := err.(strategySetupError); isSetupError {
 			glog.V(4).Infof("Error output:\n%s", errBuf.String())
-			fmt.Fprintf(errOut, "WARNING: cannot use %s: %v", s.String(), err.Error())
+			fmt.Fprintf(errOut, "WARNING: cannot use %s: %v\n", s.String(), err.Error())
 			continue
 		}
 		io.Copy(errOut, errBuf)
 		break
 	}
-	return err
+
+	return strategySetupError("No available strategies to copy to container.")
```

Comment:
so this change is causing the test failure.  you're now always returning a failure message instead of a nil error when things are ok.

```
Running test/end-to-end/core.sh:288: executing 'oc rsync examples/sample-app frontend-1-73dal:/tmp' expecting success...
FAILURE after 0.428s: test/end-to-end/core.sh:288: executing 'oc rsync examples/sample-app frontend-1-73dal:/tmp' expecting success: the command returned the wrong error code
Standard output from the command:
sending incremental file list
sample-app/
sample-app/README.md
sample-app/application-template-custombuild.json
sample-app/application-template-dockerbuild.json
sample-app/application-template-pullspecbuild.json
sample-app/application-template-stibuild.json
sample-app/cleanup.sh
sample-app/container-setup.md
sample-app/github-webhook-example.json
sample-app/pullimages.sh
sample-app/logs/
sample-app/logs/.gitkeep

sent 92822 bytes  received 210 bytes  62021.33 bytes/sec
total size is 92073  speedup is 0.99

Standard error from the command:
error: No available strategies to copy to container.
```